burotel/indico_burotel/plugin.py

```diff
@@ -80,3 +80,10 @@ def _before_request(self):
                 data = patch_time(request.json)
                 request.data = request._cached_data = json.dumps(data)
                 request._cached_json = data
+
+        if request.endpoint in ['rooms_new.create_booking', 'rooms_new.update_booking']:
```

Comment:
not needed when updating, we have the default reason there - so we can just leave the field required there